packages/webpack-plugin/src/index.js

```diff
@@ -28,19 +29,36 @@ class LoadablePlugin {
       chunkGroupChildren: true,
       hash: true,
       ids: true,
+      modules: true,
       outputPath: true,
       publicPath: true,
+      reasons: true,
     })
 
     stats.generator = 'loadable-components'
 
-    // we don't need all chunk information, only a type
-    stats.chunks = [...compilation.chunks].map(chunk => {
-      return {
+    const { sharedConsumes, sharedModules } = collectSharedModuleConsumptionStats(stats.modules);
+    delete stats.modules;
+
+    stats.sharedModules = sharedModules;
+    // stats.sharedConsumes = sharedConsumes;
+
+    stats.chunks = [];
+    [...compilation.chunks].forEach((chunk) => {
```

Comment:
we are asking `stats` without `chunks`. Why it's not empty?

Reply:
That's not originally really my line, I just changed it from map to forEach. Chunks are taken straight from compilation, not from stats (most likely to prevent adding additional stats generation time)